atomicfu-gradle-plugin/build.gradle

```diff
@@ -24,76 +24,12 @@ dependencies {
     // Atomicfu compiler plugin dependency will be loaded to kotlinCompilerPluginClasspath
     // Atomicfu plugin will only be applied if the flag is set kotlinx.atomicfu.enableJsIrTransformation=true
     compileOnly "org.jetbrains.kotlin:atomicfu:$kotlin_version"
+    runtimeOnly "org.jetbrains.kotlin:atomicfu:please-add-dependency-in-your-project"
```

Comment:
Maybe it's worth specifying some min viable, but existing version? I guess, more up-to-date version, if presented, should be preferred to it during dependencies resolution, but it also should help with all those failure in CI.

Reply:
Right, this seems like the only solution. Otherwise I can not avoid `atomicfu-gradle-plugin:runtimeClasspath` resolution.